src/mediumComplexityProblem1/IncomeTaxCalculation.java

```diff
@@ -0,0 +1,42 @@
+package mediumComplexityProblem1;
+
+import java.util.Scanner;
+
+public class IncomeTaxCalculation {
+    private double tax;
+    private double it;
+
+    public void findIncomeTax() {
+        Scanner sc = new Scanner(System.in);
+        System.out.println("Enter income :");
```

Comment:
try to use switch case.

Reply:
Switch only takes constant values in it's cases. we cannot add expressions in cases which evaluate run time.
So, can't use switch case here.

Reply:
@amitsahu8 @rohitlamsonge66  I have implemented Switch Case, Instead of if-else statements, so please review it and let me know if any changes required.